air/src/air/context.rs

```diff
@@ -237,6 +237,16 @@ impl<B: StarkField> AirContext<B> {
         self.num_transition_exemptions
     }
 
+    /// Returns the a tuple containing the main and auxiliary transition constraint degrees.
+    pub fn transition_constraint_degrees(
+        &self,
+    ) -> (&[TransitionConstraintDegree], &[TransitionConstraintDegree]) {
```

Comment:
We should return just a single vector here. The time in would take to build this vector is negligible and it would provide a cleaner interface.

Reply:
Done!